app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError
+from app.errors import VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The function returns immediately when a `VaccineError` is caught, which prevents checking all friends. Modify the logic to ensure all friends are checked, even if one is not vaccinated. Consider using a flag or counter to track the vaccination status of all friends.